src/ast.js

```diff
@@ -1,4 +1,4 @@
-const astnodes = require('../gen/astnodes.js');
+var astnodes = require('../gen/astnodes.js');
```

Comment:
this require doesn't exist in a browser. I think it would be best if astnodes were exported with `goog.exportSymbol` and loaded into the global scope like the rest. It might be best if we rename them to Sk.astnodes.

Reply:
I think that would fix my current problem.  This makes it such that I need a bundler, but the closure library doesn't work with them.  Let me do that.

Reply:
This gets me further, and I think it is the right thing to do for this first step.  But now I have a closure compiler error on a line that I didn't touch.  Investigating, but unsure of what happened.  I committed and pushed anyway.

Reply:
Never mind I figured it out.